pkg/webhook/admission/deployment/downloader_injector.go

```diff
@@ -0,0 +1,105 @@
+/*
+Copyright 2019 kubeflow.org.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+	http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package deployment
+
+import (
+	"strings"
+
+	"github.com/kubeflow/kfserving/pkg/constants"
+
+	appsv1 "k8s.io/api/apps/v1"
+	v1 "k8s.io/api/core/v1"
+)
+
+const (
+	defaultMountName         = "kfserving-download-location"
+	defaultMountPath         = "/mnt"
+	userContainerName        = "user-container"
+	downloadContainerImage   = "kcorer/kfdownloader"
```

Comment:
Need to figure out how to fix this...? maybe put it into a config-map and then somehow do some ko like thing for the dev loop?

Reply:
agree, I think in kfserving release  we should add a kfserving configmap about configuration such as downloadContainerImage image, TensorflowServingImageName +DefaultTensorflowRuntimeVersion and so on, and in manager deployment adds related env whose value refers the configmap counterpart.
So I suggest `buildInitContainer` should try to get `downloadContainerImage` by ENV, if doesn't exist, use `kcorer/kfdownloader:latest` then.
Thus, in each release, we only need update the released configmap entry without updating our source code.

Reply:
we already have ConfigMap, we can add the download image there.

Reply:
Will do as a separate commit, tracked by #189